exporter/opentelemetry-exporter-otlp-proto-common/src/opentelemetry/exporter/otlp/proto/common/_internal/metrics_encoder/__init__.py

```diff
@@ -145,12 +150,16 @@ def _common_configuration(
                     otel_exporter_otlp_metrics_default_histogram_aggregation,
                 )
 
-            histogram_aggregation_type = ExplicitBucketHistogramAggregation
+            instrument_class_aggregation = {
+                Histogram: ExplicitBucketHistogramAggregation(),
+            }
+
+        instrument_class_aggregation.update(preferred_aggregation or {})
 
         MetricExporter.__init__(
```

Comment:
This function is getting rather large now. What do you think about breaking most of it up into two functions, kind of like this:

```
        MetricExporter.__init__(
            self,
            preferred_temporality=self.get_temporality(preferred_temporality),
            preferred_aggregation=self.get_aggregation(preferred_aggregation),
        )
```

Reply:
Great idea @pmcollins, fixed on 33ec97a